spec/support/templates/user_decorator.rb

```diff
@@ -0,0 +1,18 @@
+require 'draper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.